### PR TITLE
Meenu new sign up page rebranding

### PR DESCRIPTION
**Pull request by meenakshi-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/components/custom/_signup-new.tsx [hunk 1]
```diff
@@ -335,7 +238,9 @@ const SignupNew = ({
                     social
                 >
                     <img src={Google} alt="google" width="24" height="24" />
-                    <SocialText>Google</SocialText>
+                    <SocialText as="p" align="center">
+                        <Localize translate_text="Google" />
```
Comment (NikitK-deriv) [inline src/components/custom/_signup-new.tsx:242]: Are we exactly have to translate name of companies?

File: src/components/custom/_signup-new.tsx [hunk 2]
```diff
@@ -344,9 +249,12 @@ const SignupNew = ({
                     id="dm-signup-facebook"
                     type="button"
                     social
+                    bgColor="var(--color-blue-15)"
```
Comment (NikitK-deriv) [inline src/components/custom/_signup-new.tsx:252]: We usually using snake_case for props, let's rename it a little bit
```suggestion
                    bg_color="var(--color-blue-15)"
```

File: src/components/custom/signup.tsx [hunk 3]
```diff
@@ -47,18 +47,27 @@ const Form = styled.form<FormProps>`
...
-const ResponseWrapper = styled.div`
+const ResponseWrapper = styled(Flex)`
     justify-content: center;
-    max-width: 33rem;
+    max-width: 24rem;
     margin: 0 auto;
     flex-direction: column;
     padding: 2rem 1rem;
+    gap: 12px;
+    margin-top: -100px;
```
Comment (NikitK-deriv) [inline src/components/custom/signup.tsx:57]: Can we set `margin-bottom: -100px;` instead?

File: src/components/custom/signup.tsx [hunk 4]
```diff
@@ -47,18 +47,27 @@ const Form = styled.form<FormProps>`
...
 
-const ConfirmationMessage = styled.div`
+const ConfirmationMessage = styled(Header)`
     text-align: center;
     font-size: 16px;
     word-wrap: break-word;
+    line-height: 24px;
+    @media ${device.mobileL} {
+        font-size: 14px;
+    }
```
Comment (NikitK-deriv) [inline src/components/custom/signup.tsx:70]: Properties `text-align, font-size, line-height` already exist in `Header` via passing `type`. Are we need to reach special view here?

File: src/components/custom/signup.tsx [hunk 5]
```diff
@@ -203,26 +220,16 @@ const Signup = (props: SignupProps) => {
                 return <SignupDefault {...parameters}></SignupDefault>
         }
     }
+
     return props.submit_state === 'success' ? (
         <ResponseWrapper>
-            <Header as="h3" type="section-title" align="center" weight="normal">
+            <Header as="h3" type="subtitle-1" align="center" weight="700">
```
Comment (NikitK-deriv) [inline src/components/custom/signup.tsx:226]: Header's weight == bold == 700 by default

Comment (Meenusiju) [reply src/components/custom/signup.tsx]: Thanks for your comments @NikitK-deriv , fixed everything in new commit.

**Pull request closed without merging**